examples/src/test/kotlin/DatabaseCollectionsTest.kt

```diff
@@ -1,19 +1,19 @@
 
-import com.mongodb.client.model.CreateCollectionOptions
-import com.mongodb.client.model.Filters
-import com.mongodb.client.model.ValidationOptions
+import com.mongodb.client.model.*
 import com.mongodb.kotlin.client.coroutine.MongoClient
 import config.getConfig
 import kotlinx.coroutines.flow.toList
 import kotlinx.coroutines.runBlocking
 import org.bson.codecs.pojo.annotations.BsonId
 import org.bson.types.ObjectId
 import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.Assertions
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.TestInstance
+import java.util.concurrent.TimeUnit
+import kotlin.test.Ignore
```

Comment:
is this actually being used?

Reply:
WIll remove any unused imports from both code files relating to this PR